experiments/standalone/Soil/full_soil_implicit.jl

```diff
@@ -0,0 +1,238 @@
+using ClimaCore
```

Comment:
maybe we could just title this file "full_soil.jl" or "energy_hydrology.jl"? The "implicit" is kind of implied since that's how this model will always be stepped